lib/balance-tx/test/spec/Internal/Cardano/Write/Tx/BalanceSpec.hs

```diff
@@ -2771,17 +2672,22 @@ instance Arbitrary (PartialTx Write.BabbageEra) where
             mempty
     shrink (PartialTx tx inputUTxO redeemers timelockKeyWitnessCounts) =
         [ PartialTx tx inputUTxO' redeemers timelockKeyWitnessCounts
-        | inputUTxO' <- shrinkInputResolution @Write.BabbageEra inputUTxO
+        | inputUTxO' <- shrinkInputResolution inputUTxO
         ] <>
         [ restrictResolution $
             PartialTx
                 (fromCardanoApiTx tx')
                 inputUTxO
                 redeemers
                 timelockKeyWitnessCounts
-        | tx' <- shrinkTxBabbage (toCardanoApiTx tx)
+        | tx' <- shrinkCardanoApiTx (toCardanoApiTx tx)
         ]
 
+      where
+        shrinkCardanoApiTx = case recentEra @era of
+            RecentEraBabbage -> shrinkTxBabbage
+            RecentEraConway -> \_ -> [] -- no shrinker implemented yet
```

Comment:
If we intend to do this in future, then we should probably create a ticket for this:
```suggestion
            RecentEraConway -> \_ ->
                -- TODO: APD-xxxx
                -- Implement shrinker for conway
                []
```

Reply:
I think this is in "not urgent, not important" square of the Eisenhower matrix:

<img width="574" alt="Skärmavbild 2024-01-26 kl  14 08 27" src="https://github.com/cardano-foundation/cardano-wallet/assets/304423/7e89b0c5-f560-40c9-865c-37ea7bf46625">
ing